manifests/apache.pp

```diff
@@ -62,8 +62,6 @@
       owner => $::apache::user,
       group => $::certs::group,
       mode  => '0440',
-    } ->
```

Comment:
Are you sure this is correct? Isn't there a chance the key will have incorrect permissions for apache? https://github.com/Katello/puppet-certs/pull/137 does solve it by containing the keypair in a define and let that notify apache.